api/go.mod

```diff
@@ -8,14 +8,14 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
-	google.golang.org/grpc v1.41.0
```

Comment:
could you put those three picks into one PR under OCPBUGS-21221, so it's tracked correctly?

* https://github.com/etcd-io/etcd/pull/16739
* https://github.com/etcd-io/etcd/pull/16625
* https://github.com/etcd-io/etcd/pull/16781

That should be it, @dusk125 :)